tests/python/test_build_data.py

```
@@ -597,6 +597,48 @@ def fake_scrape_place(url: str, **_: object) -> SimpleNamespace:
+        place = RawPlace(
+            name="Bilmonte",
+            address=None,
+            maps_url="https://www.google.com/maps/search/?api=1&query=Bilmonte",
+            cid="1343378048703211865",
+            lat=41.3894089,
+            lng=2.1636435,
+        )
+        captured_request_body: dict[str, object] = {}
+
+        class FakeResponse:
+            def __enter__(self) -> "BuildDataTests.test_fetch_places_api_enrichment_uses_city_country_bias_for_name_only_search_urls.<locals>.FakeResponse":
```

Ruff (0.15.11)

[error] Syntax error in forward annotation: Expected an identifier

(F722)